forte/mrdsrg-spin-integrated/master_mrdsrg.cc

```diff
@@ -250,17 +251,56 @@ void MASTER_DSRG::fill_density() {
 void MASTER_DSRG::init_fock() {
     outfile->Printf("\n    Filling Fock matrix from ForteIntegrals ......... ");
     Fock_ = BTF_->build(tensor_type_, "Fock", spin_cases({"gg"}));
-    Fock_.iterate([&](const std::vector<size_t>& i, const std::vector<SpinType>& spin, double& value) {
-        if (spin[0] == AlphaSpin) {
-            value = ints_->get_fock_a(i[0], i[1]);
-        } else {
-            value = ints_->get_fock_b(i[0], i[1]);
-        }
-    });
+
+    bool JKSafe = foptions_->get_bool("EMBEDDING_JKFOCK");
+
+    if (!JKSafe) {
+        outfile->Printf("\n    Debug #2");
+        build_custom_fock();
```

Comment:
Wait, are you overriding a Fock from somewhere else? Again, what code is responsible for building Fock?